backend/app/models.py

```diff
@@ -570,6 +578,8 @@ class DirectUploadAppDeveloper(Base):
     )
     is_primary = Column(Boolean, nullable=False)
 
+    is_invite_pending = Column(Boolean, nullable=False, server_default=text("false"))
```

Comment:
I'm actually wondering, if we should manage that in a different (but mostly identical) table. As that would decrease the likelihood of giving wrong access when we change stuff up.

It also might allow us, to lock down the table constraints better.

Reply:
Good idea. It might also make it easier to implement ownership transfer.